.github/scripts/ai_review/review_pr.py

```
@@ -0,0 +1,579 @@
+        notes,
+    )
+
+
+class GeminiRateLimitError(RuntimeError):
+    pass
+
+
+def _is_gemini_rate_limited(status_code: int, body_text: str) -> bool:
+    if status_code == 429:
+        return True
+    # 念のため、JSONエラーのstatusも見る（RESOURCE_EXHAUSTED など）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)